### PR TITLE
Avoid negative or zero values as quantity

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | When a product inside a wishlist has been added to the cart, reset its quantity to 1 instead of zero.
| Type?         | bug fix
| BC breaks?    | Nope
| Deprecations? | Nope
| Fixed ticket? | /
| How to test?  | Adding a product in the cart from the wishlist page updates its quantity to 1.

### CODE REVIEW

File: classes/WishList.php [hunk 1]
```diff
@@ -425,9 +425,16 @@ public static function addBoughtProduct($id_wishlist, $id_product, $id_product_a
             return false;
         }
 
+        $newQuantity = (int) $result['quantity'] - (int) $quantity;
+
+        // Avoid negative or zero values
+        if ($newQuantity < 1) {
+            $newQuantity = 1;
```
Comment (NeOMakinG) [inline classes/WishList.php:432]: What if the default quantity of the product is something else than 1 ?

Comment (Quetzacoalt91) [reply classes/WishList.php:432]: I'm looking for this configuration data, do you know where I can find it?

Comment (NeOMakinG) [reply classes/WishList.php:432]: Mmmh, at least on FO when I receive your datas, I've a minimal_quantity key

Comment (Quetzacoalt91) [reply classes/WishList.php:432]: oh okay, that a piece of information to be defined per product.I'll have a look.

Comment (Quetzacoalt91) [reply classes/WishList.php]: PR has been updated to use the minimal quantity

**Pull request merged**